themes/ddbasic/template.php

```diff
@@ -460,6 +460,12 @@ function ddbasic_menu_link__menu_tabs_menu($vars) {
 
   $element['#localized_options']['attributes']['class'][] = 'js-topbar-link';
 
+  // Some links are not translated properly, this makes sure these links are
+  // run through the t function.
+  if ($element['#original_link']['title'] == $element['#original_link']['link_title']) {
+    $element['#title'] = t($element['#title']);
+  }
+
```

Comment:
This seems like the wrong way to go about this. I cannot spot anywhere in the codebase containing the text "Log out" which is not wrapped in `t()`. I do not know what process went through arriving at this solution but without further information about the issue I cannot accept this.

The previous version of DDBasic did not need this.